CHANGELOG.md

```diff
@@ -7,7 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
 * Add description_html option to notice component (PR #740)
+* Add `autocomplete` attribute option to input component and use it on feedback component's survey signup form (PR #733)
```

Comment:
This should literally be the name of the PR, see my previous comment.

Reply:
I've rebased but instead of adjusting the changelog entry, I changed the name of the PR. ;-)